### ISSUE TITLE
Unable to add new accounts

### ISSUE THREAD

**Issue by zmiguel** [GoldenGnu/jeveassets#105]
It's been a few months since I used this, opened it and it updated to **version 6.0.3**.
Tried to do a data update on *update > update* all selected as is usual.

Price data loads fine, however, **Accounts** and **Data** get 401 and 403 errors, respectively.

I've tried to re-auth the accounts but it seems the web server doesn't start to receive the SSO data, I get an *ERR_EMPTY_RESPONSE* on chrome and the app keeps saying *Validating API Key*

If there's any other information you need just let me know.

Thanks!

**Comment by zmiguel** [comment]
Error log entry for account import sequence

```INFO: [AWT-EventQueue-0] net.nikr.eve.jeveasset.gui.shared.components.JDialogCentered - Hiding: Account Import Dialog
INFO: [AWT-EventQueue-0] net.nikr.eve.jeveasset.gui.shared.components.JDialogCentered - Showing: Account Import Dialog
INFO: [AWT-EventQueue-0] net.nikr.eve.jeveasset.io.shared.DesktopUtil - Browsing: https://login.eveonline.com/v2/oauth/authorize?response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A2221&client_id=<redacted>&scope=<scopes>&state=jeveassets&code_challenge99smoI_FwpHsiju9W<redacted>k=&code_challenge_method=S256
ERROR: [Thread-4] net.nikr.eve.jeveasset.io.esi.MicroServe - Protocol family not supported
java.net.SocketException: Protocol family not supported
	at java.base/java.net.PlainSocketImpl.socketAccept(PlainSocketImpl.java:180)
	at java.base/java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:458)
	at java.base/java.net.ServerSocket.implAccept(ServerSocket.java:556)
	at java.base/java.net.ServerSocket.accept(ServerSocket.java:524)
	at net.nikr.eve.jeveasset.io.esi.MicroServe$ConnectionListener.run(MicroServe.java:104)```

**Comment by GoldenGnu** [comment]
The fix for this have been released in 6.0.4
If you could confirm it's fixed, that would be wonderful

**Comment by zmiguel** [comment]
Confirming it's working with 6.0.4 update.

Thanks!
